File: 프로그래머스/0/120815. 피자 나눠 먹기 （2）/피자 나눠 먹기 （2）.js
function solution(n) {
    let answer = 0;
    let a = 1;
    let b = 1;
    while (true) {
        if (n === 6) {
            answer += 1;
            break;
        }
        if (n * a <= 6 * b) {
            a += 1;
        } else {
            b += 1;
            answer += 1;
        }
        if (n * a === 6 * b) {
            answer += 1;
            break;
        }
    }
    return answer;
}

File: 프로그래머스/0/120815. 피자 나눠 먹기 （2）/피자 나눠 먹기 （2）.c
#include <stdio.h>

int solution(int n) {
    int answer = 0;
    int a = 1;
    int b = 1;

    while (1) {
        if (n == 6) {
            answer += 1;
            break;
        }
        if (n * a <= 6 * b) {
            a += 1;
        } else {
            b += 1;
            answer += 1;
        }
        if (n * a == 6 * b) {
            answer += 1;
            break;
        }
    }

    return answer;
}